src/backend/launcher.ts

```diff
@@ -121,6 +124,92 @@ async function prepareLaunch(
     }
   }
 
+  if (
+    (gameSettings.gamescope?.enableLimiter ||
+      gameSettings.gamescope?.enableUpscaling) &&
+    !isSteamDeckGameMode
+  ) {
+    const gameScopeBin = await searchForExecutableOnPath('gamescope')
+    if (!gameScopeBin) {
+      return {
+        success: false,
+        failureReason:
+          'Gamescope is enabled, but `gamescope` executable could not be found on $PATH'
+      }
+    }
```

Comment:
I wonder if we should puts something in the logs and ignore gamescope instead of failing?

because if a user uninstalls gamescope and a game has it configured or it's configured in the defaults, there's no way for the user to disable gamescope form Heroic because we show `We could not found gamescope on the PATH. Install it or add it to the PATH.` and hide the gamescope settings.

Or maybe we can always show the gamescope settings but show a warnings at the beginning of that section saying the executable was not found, so users can disable it.

Reply:
Heroic should check for gamescope at launch as well, so it can ignore the gamescope settings if was not found.
This would avoid breaking some games in this example.

But I also vote for the disabled setting instead of removing it, so we can be more clear.

Reply:
yeah disabling the feature if no gamescope found is the best way in my opinion

Reply:
just to clarify, currently the feature is partially disabled when the binary is not found (it shows a message and no actual settings)

that is what would cause this issue: if you had that available, enabled it, and then uninstall gamescope, the settings are disabled but the game would fail to run and we have no way of disabling it since the settings are gone

so we have to either:
- not fail if it's enabled but not available
- not hide the settings if it's enabled, even if not available to allow disabling it (we could hide the settings if disabled and not available to prevent enabling it)
- do both things to be extra safe